examples/intro_to_observations.jl

```diff
@@ -0,0 +1,145 @@
+                                        closure = closure)
+                                        
+    set!(model, b = (x, y, z) -> N² * z)
+    
+    simulation = Simulation(model; Δt, stop_time=12hours)
+
+    init_with_parameters(file, model) = file["parameters"] = (; Qᵇ, Qᵘ, Δt)
+    
+    simulation.output_writers[:fields] = JLD2OutputWriter(model, merge(model.velocities, model.tracers),
+                                                          schedule = TimeInterval(4hour),
+                                                          prefix = name,
+                                                          array_type = Array{Float64},
+                                                          field_slicer = nothing,
+                                                          init = init_with_parameters,
+                                                          force = true)
+    
+    run!(simulation)
+
+    return data_path
+end
+
+# and invoke it:
+
+data_path = generate_free_convection_synthetic_observations()
+
+# # Loading observations
+#
+# When synthetic observations are constructed from simulation data, we
+# can select
+#
+# * The fields to include via `field_names`
+#
+# * Which data in the time-series to include via the `times` keyword.
+#   This can be used to change the initial condition for a calibration run.
+#
+# ## Loading observations with a single field:
+
+observations = OneDimensionalTimeSeries(data_path, field_names=:b, normalize=ZScore)
+
+# ## Loading observations with two fields:
+
+observations = OneDimensionalTimeSeries(data_path, field_names=(:u, :b), normalize=ZScore)
+
+# ## Loading observations with specified times
+
+times = observations.times[2:end]
+observations = OneDimensionalTimeSeries(data_path, field_names=(:u, :b), normalize=ZScore, times=times)
+
+# # Visualizing observations
+
+# For this we include the initial condition and ``v`` velocity component,
+
+observations = OneDimensionalTimeSeries(data_path, field_names=(:u, :v, :b), normalize=ZScore)
+
+fig = Figure()
+
+ax_b = Axis(fig[1, 1], xlabel = "Buoyancy [m s⁻²]")
+ax_u = Axis(fig[1, 2], xlabel = "Velocities [m s⁻¹]")
+
+z = znodes(Center, observations.grid)
+
+colorcycle = [:black, :red, :blue, :orange]
+
+for i = 1:length(observations.times)
+    b = observations.field_time_serieses.b[i]
+    u = observations.field_time_serieses.u[i]
+    v = observations.field_time_serieses.v[i]
+    t = observations.times[i]
+
+    label = "t = " * prettytime(t)
+    u_label = i == 1 ? "u, " * label : label
+    v_label = i == 1 ? "v, " * label : label
+
+    lines!(ax_b, interior(b)[1, 1, :], z; label, color=colorcycle[i])
+    lines!(ax_u, interior(u)[1, 1, :], z; linestyle=:solid, color=colorcycle[i], label=u_label)
+    lines!(ax_u, interior(v)[1, 1, :], z; linestyle=:dash, color=colorcycle[i], label=v_label)
+end
+
+axislegend(ax_b, position=:rb)
+axislegend(ax_u, position=:lb, merge=true)
+
+save("intro_to_observations.svg", fig)
```

Comment:
```suggestion
save("intro_to_observations.svg", fig); nothing #hide 
```

Reply:
I'm not a fan of all the hiding, but let's see what it looks like

Reply:
One need not know that docs need to save a .svg file and display it as an image in the html. That's quite an esoteric detail, no? The users should just be able to copy all the code lines from the docs in their repl and see a figure. Which they are, right?

Reply:
<img width="986" alt="Screen Shot 2021-11-11 at 4 38 04 pm" src="https://user-images.githubusercontent.com/7112768/141243645-19374895-ac13-4ac1-8748-f96fa3b1cfc0.png">

So all these `CairoScreen` mumbo jumbo is definitely not needed.

This
```julia
save("intro_to_observations.svg", fig)
nothing #hide 
```
will still show `save(...)` in the code but preclude printing out the `CairoScreen{...`. Otherwise what I suggested above also hides out the `save(...)`.